Makefile

```diff
@@ -17,20 +16,27 @@ capture.so: capture.c
 	sed 's/@PREFIX@/$(subst /,\/,$(PREFIX))/' $+ > $@
 
 install: all
-	install -d $(DESTDIR)$(PREFIX)/bin
-	install -d $(DESTDIR)$(PREFIX)/share/adi_colorimeter/
-	install -d $(DESTDIR)$(PREFIX)/lib/adi_colorimeter/
-	install ./org.adi.pkexec.adi_colorimeter.policy /usr/share/polkit-1/actions 
-	install ./adi_colorimeter $(DESTDIR)$(PREFIX)/bin/
-	install ./capture.so $(DESTDIR)$(PREFIX)/lib/adi_colorimeter/
-	install ./adi_colorimeter.glade $(DESTDIR)$(PREFIX)/share/adi_colorimeter/
-	./setup.py install --prefix=$(PREFIX) --root=$(DESTDIR)
+	install -d $(PREFIX)/bin
```

Comment:
Fix indentation and paths should be also in a separate commit than the one that adds uninstall.